libraries/protobuf/serialization/build.sh

```diff
@@ -14,7 +12,7 @@ cmake .. -DCMAKE_BUILD_TYPE=Release
 cmake --build .
 
 bin/sensor
-touch __init__.py
 
-popd
-python main.py
+pip install protobuf
+python ../main.py
+pip uninstall -y protobuf
```

Comment:
I'm not able to make it work with a virtualenv 😢 , so I use the main python environment and uninstall the new package afterward

Reply:
protobuf is evil :smiling_imp:

Reply:
why don't you remove all python package installed by protobuf as last step? So I think is possible to use with virtualenv

Reply:
the problem with the virtualenv is that I'm not able to install `protobuf` inside the virtualenv in Linux (like I do in Windows or Mac), and to pass this new check (--- tests do not modify the python environment ---) I need to remove it here

Reply:
Windows has failed too 😢 ...so it is something related to the levels of bash calling bash calling python calling bash... so I've moved to the `uninstall` strategy